action-server/covidflow/utils/testing_locations.py

```diff
@@ -25,6 +27,32 @@
 }
 
 
+class Day(Enum):
+    monday = 1
+    tuesday = 2
+    wednesday = 3
+    thursday = 4
+    friday = 5
+    saturday = 6
+    sunday = 7
+
+
+class OpeningHour(NamedTuple):
```

Comment:
Maybe `OpeningPeriod` or `OpeningInterval`? `OpeningHour` makes it sound like it's only one hour or the hour at which it opens.